akka-docs/rst/scala/dispatchers.rst

```diff
@@ -153,6 +130,10 @@ And then using it:
 
 .. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#defining-fixed-pool-size-dispatcher
 
+Another example that uses the thread pool based on the number of cores (e.g. for CPU bound tasks)
+
+.. includecode:: ../scala/code/docs/dispatcher/DispatcherDocSpec.scala#my-thread-pool-dispatcher-config
```

Comment:
This is at the bottom of the page in the section "More dispatcher configuration examples". There we also show the `fixed-pool-size = 16`, and at the top of the page we now show the `fixed-pool-size = 16` example.